test/extensions/reset_password/ecto/context_test.exs

```diff
@@ -15,6 +15,10 @@ defmodule PowResetPassword.Ecto.ContextTest do
       assert Context.get_by_email("test@example.com", @config)
       assert Context.get_by_email("TEST@EXAMPLE.COM", @config)
     end
+
+    test "email is trimmed when it's the user id field" do
+      assert Context.get_by_email(" test@example.com ", @config)
```

Comment:
I think this is good, but I'll have to check if this is the proper way of handling user id field before merging this PR.